docs/install-run/special/linux.txt

```diff
@@ -23,19 +23,13 @@ Linux`_.
 One-Step Setup
```

Comment:
Should we maybe change the title as well? Because the try script won't setup any installation...

Reply:
Any suggestions for the wording? I kept it - right now it is consistent with the `mac.txt` which already referred to the `try.crate.io` script.

Reply:
ok, I have no good suggestion anyway